build.sc

```diff
@@ -23,10 +14,19 @@ import mill.scalalib.publish._
 import mill.util.Jvm
 import mill.resolve.SelectMode
 import mill.contrib.buildinfo.BuildInfo
-import mill.scalalib.api.Versions
 import mill.T
 import mill.define.Cross
 
+// plugins and dependencies
+import $file.ci.shared
+import $file.ci.upload
+import $ivy.`org.scalaj::scalaj-http:2.4.2`
+import $ivy.`de.tototec::de.tobiasroeser.mill.vcs.version::0.4.0`
+import $ivy.`com.github.lolgab::mill-mima::0.1.1`
+import $ivy.`net.sourceforge.htmlcleaner:htmlcleaner:2.29`
+import $ivy.`com.lihaoyi::mill-contrib-buildinfo:`
+import $ivy.`com.goyeau::mill-scalafix::0.4.0`
```

Comment:
I moved these after the ordinary imports to work around IntelliJ removing them when "organizing" them. Since we parse them ahead of compilation, it makes no semantic difference.